frontend/www/js/omegaup/components/problem/Collection.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <h1 class="card-title">{{ T.collectionTitle }}</h1>
+  </div>
```

Comment:
```suggestion
  <div class="card">
    <div class="card-body">
      <h5 class="card-title">{{ T.collectionTitle }}</h5>
    </div>
  </div>
```

se puede usar la plantilla de la [documentación](https://getbootstrap.com/docs/4.5/components/card/#titles-text-and-links)? o si se desea tener este título separado se puede usar [`card-header`](https://getbootstrap.com/docs/4.5/components/card/#titles-text-and-links) en vez de `card-title`.